src/components/DataCollector/activity/click.js

```diff
@@ -29,14 +41,16 @@ const createClickHandler = (window, logger, collect) => {
       }
     }
     if (linkUrl && isSupportedAnchorElement(clickedObj)) {
+      const linkType = determineLinkType(window, config, linkUrl, clickedObj);
+      const linkName = "Link Click";
       // TODO: Update name (link name) and support exit, other, and download link types
       collect({
```

Comment:
Do we want to automatically set `documentUnloading` here based on the link type?

Reply:
I like that!

Reply:
Yes, good point